source/crud/tests/README.rst

```diff
@@ -54,24 +42,19 @@ Each YAML file has the following keys:
 
   - ``description``: The name of the test.
 
-  - ``operation``: Document describing the operation to be executed. This will
-    have the following fields:
+  - ``operations``: Array of documents, each describing an operation to be
+    executed. Each document has the following fields:
 
-    - ``name``: The name of the operation as defined in the specification. The
-      name `db-aggregate` refers to database-level aggregation.
```

Comment:
@kvwalker: Tagged you on review for this bit. I think this was a leftover and can be safely removed.

Reply:
Yep, this can be removed.